transfromage/libs/api/Client/Listener/Modern/player/cheese.lua

```diff
@@ -6,7 +6,7 @@ local onPlayerGetCheese = function(self, packet, connection, identifiers)
 	local player = self.playerList[packet:read32()]
 	if not player then return end
 
-	player.hasCheese = packet:readBool()
+	player.hasCheese = packet:read8() > 0
```

Comment:
Since it's 8b (up to 0xFF) then it can only be the number of cheese on the mouse ([0-3]).

Please add both fields

```Lua
player.cheese = packet:read8()
player.hasCheese = player.cheese > 0
```